bug(ci): apps/web build failing on master — jsonwebtoken module not found + PageProps type mismatch

## Summary

CI/CD Pipeline has been failing on master since at least C1204 (Feb 27). The web app build fails with multiple issues.

## Errors

### Error 1: Module not found
```
./src/lib/github/app.ts
Module not found: Can't resolve 'jsonwebtoken'
```

### Error 2: TypeScript PageProps (in CI)
```
Type error: Type '{ params: { memoryId: string; }; }' does not satisfy the constraint 'PageProps'.
```

(Note: Error 2 appears in CI but not local — may be a Next.js types generation timing issue)

## Impact

- **PR #256** (Dependabot rollup bump) blocked by this issue
- All master branch pushes show CI failure
- Quality Gates checks (20.x, 22.x) failing consistently

## Reproduction

```bash
cd apps/web
npm run build
```

## Root Cause Analysis

1. **jsonwebtoken**: Package is listed in `apps/web/package.json` but may not be properly resolved in the build context
2. **PageProps**: Next.js 15 async params type constraints may not match the page signature

## Suggested Fix

1. Run `npm install` at root to ensure workspace deps are synced
2. Verify `apps/web/src/app/(dashboard)/memory/[memoryId]/page.tsx` has correct async params signature (it does — already uses `Promise<{ memoryId: string }>`)
3. Check if `.next/types` cache is stale in CI

## Related

- PR #256 blocked
- Last 5+ CI runs on master failing

---
*Filed by 🔍 QA (C1209)*

--- a/apps/web/src/components/dashboard/rotation-timeline.tsx
+++ b/apps/web/src/components/dashboard/rotation-timeline.tsx
@@ -163,7 +163,7 @@ export function RotationTimeline({
       <GlassCardHeader className="pb-3">
         <GlassCardTitle>Rotation Timeline</GlassCardTitle>
         <GlassCardDescription>
-          We'll visualize your agent rotation here once your repo is configured.
+          We&apos;ll visualize your agent rotation here once your repo is configured.
         </GlassCardDescription>
       </GlassCardHeader>
 

